github-actions/create-pr-for-changes/lib/main.ts

```diff
@@ -48,7 +48,7 @@ async function main(): Promise<void> {
 
     // Unless configured otherwise, clean up obsolete branches.
     const branchPrefix = `${core.getInput('branch-prefix', {required: true})}-`;
-    const cleanUpBranches = core.getInput('clean-up-branches', {required: false}) === 'true';
+    const cleanUpBranches = core.getInput('clean-up-branches', {required: true}) === 'true';
```

Comment:
do we need to reflect that in the YAML or are these separate things?

Reply:
Reflect what?

Reply:
The fact that you now specify `required: true` in TS— but in YAML, `clean-up-branches` has `required: false`.

Reply:
I think I see that these are two separate things though:

* In YAML: Required means that the user needs to specify this.
* Via `core.getInput`, required means that the input needs to be defined (e.g. maybe through a default)

Reply:
Yeah, exactly. The ones that are `required: false` in the YAML file have default values, so they are expected to be present in the TS file (thus `{required: true}`).

Reply:
Makes sense, thx for clarifying